File: pyjl/cases/built_ins.py
#!/usr/bin/env python3


def default_builtins():
    a = str()
    b = bool()
    c = int()
    assert a == ""
    assert b == False
    assert c == 0


if __name__ == "__main__":
    a = max(1, 2)
    print(a)
    b = min(1, 2)
    print(b)

File: pyjl/tests/tests_py2many-py2many/built_ins.jl
function default_builtins()
a = string()
b = false
c = zero(Int)
@assert(a == "")
@assert(b == false)
@assert(c == 0)
end

function main()
default_builtins();
a = max(1, 2)
@assert(a == 2)
b = min(1, 2)
@assert(b == 1)
println("OK");
end

main()
